### ISSUE TITLE
Code blocks in comments

### ISSUE THREAD

**Issue by atesgoral** [lionleaf/dwitter#247]
Proposal: Use backtick (<code>\`</code>).

* Escape backticks within code as <code>\\`</code>

**Comment by sigvef** [comment]
I like it!

**Comment by whackashoe** [comment]
would hljs (https://highlightjs.org/usage/) be welcome for this?

**Comment by atesgoral** [comment]
Let's hold off on merging this. I'm not 100% sold on the idea of using `` \` ``. Adding support for both ` `` ` and ` ``` ` might be better.

**Comment by atesgoral** [comment]
Syntax highlighting would also be cool if it doesn't turn out to be too expensive computationally.

**Comment by lionleaf** [comment]
oops, I merged straight away ^^ Didn't see your request.

``` might work well.

I would imagine syntax highlighting would be done on the client side with js anyway, so shouldn't be too expensive?

**Comment by atesgoral** [comment]
@lionleaf LOL I accidentally put the comment on the issue instead of the PR. We can enhance later with new PRs.

**Comment by atesgoral** [comment]
Yeah, we can throw in a client-side syntax highlighter.

**Comment by lionleaf** [comment]
Implemented by #270 .  

Code highlighting and other follow-up work can be opened as new issues if you want @atesgoral :D
